docs/grpc-tls-howto.md

```diff
@@ -86,13 +86,13 @@ Like the contour cert, this CSR uses the file [_integration/cert-envoy.ext](./_i
 Next, we create the required secrets in the target Kubernetes cluster:
 
 ```
-kubectl create secret -n heptio-contour generic cacert --from-file=./certs/cacert.pem
-kubectl create secret -n heptio-contour tls contourcert --key=./certs/contourkey.pem --cert=./certs/contourcert.pem
-kubectl create secret -n heptio-contour tls envoycert --key=./certs/envoykey.pem --cert=./certs/envoycert.pem
+kubectl create secret -n projectcontour generic cacert --from-file=./certs/cacert.pem
+kubectl create secret -n projectcontour tls contourcert --key=./certs/contourkey.pem --cert=./certs/contourcert.pem
+kubectl create secret -n projectcontour tls envoycert --key=./certs/envoykey.pem --cert=./certs/envoycert.pem
 ```
 
 Note that we don't put the CA **key** into the cluster, there's no reason for that to be there, and that would create a security problem. That also means that the `cacert` secret can't be a `tls` type secret, as they must be a keypair.
 
 # Conclusion
 
-Once this process is done, the certificates will be present as Secrets in the `heptio-contour` namespace, as required by `examples/ds-hostnet-split`.
+Once this process is done, the certificates will be present as Secrets in the `projectcontour` namespace, as required by `examples/ds-hostnet-split`.
```

Comment:
Update to `examples/contour`?

Reply:
oooh, good catch, i'll grep for deceased examples.